apollo-client/src/main/java/com/ctrip/framework/apollo/internals/DefaultConfigManager.java

```diff
@@ -30,21 +30,30 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class DefaultConfigManager implements ConfigManager {
-  private ConfigFactoryManager m_factoryManager;
+  private final ConfigFactoryManager m_factoryManager;
 
-  private Map<String, Config> m_configs = Maps.newConcurrentMap();
-  private Map<String, ConfigFile> m_configFiles = Maps.newConcurrentMap();
+  private final Map<String, Config> m_configs = Maps.newConcurrentMap();
+  private final Map<String, ConfigFile> m_configFiles = Maps.newConcurrentMap();
+  private final Map<String, Object> m_configLocks = Maps.newConcurrentMap();
+  private final Map<String, Object> m_configFileLocks = Maps.newConcurrentMap();
 
   public DefaultConfigManager() {
     m_factoryManager = ApolloInjector.getInstance(ConfigFactoryManager.class);
   }
 
   @Override
   public Config getConfig(String namespace) {
+    String lockKey = buildLockKey(namespace);
+    Object lock = m_configLocks.get(lockKey);
+    if (lock == null) {
+      synchronized (lockKey) {
+        lock = m_configLocks.computeIfAbsent(lockKey, k -> new Object());
+      }
+    }
```

Comment:
Is it ok to simplify the logic as follows?
```suggestion
    Object lock = m_configLocks.computeIfAbsent(lockKey, k -> new Object());
```